### DISCUSSION TITLE
Missing spec

### DISCUSSION THREAD

**Discussion by SafaAlfulaij** [django-json-api/django-rest-framework-json-api#944] [General]
####  Updating a Resource’s Relationships - [200 OK](https://jsonapi.org/format/#crud-updating-responses-200)
> If a server accepts an update but also changes the resource(s) in ways other than those specified by the request (for example, updating the updated-at attribute or a computed sha), it MUST return a 200 OK response. **The response document MUST include a representation of the updated resource(s) as if a GET request was made to the request URL**.

##### Issue:
There is no way currently to tell the renderer to render the included fields even when provided in the serializer’s data.

**Comment by SafaAlfulaij** [comment]
I think that the current way the renderer works isn't correct and wrong in terms of separation of concerns.
Serializers should tell what are the final data that should be rendered. and therefore have the say in it.

Kind of related to #921

**Comment by sliverc** [comment]
I do not think the spec snippet you posted is talking about `included` resp. compound documents. It simply says if you call the same URL with `GET` again as the one you do the `PATCH` call on the response must be the same. And that is what DJA already does.

I think what works (not tested) if you add the `include` query parameters to the URL you do the `PATCH` on.
